### DISCUSSION TITLE
Global Yaml File

### DISCUSSION THREAD

**Discussion by ishaanshah-alation** [codecov/feedback#76] [Ideas]
We wanted to see if there was a way to have conditionals in the global yaml file. We want a centralized area where we can control the thresholds of each of our repositories without having to create a new commit to each of our repos. Is there a way to possible use the flag names or anything of that nature as a conditional in the global yaml file.

**Comment by rohan-at-sentry** [comment]
Hey @ishaanshah-alation 

If I understand correctly, you're looking to set coverage thresholds for all repo's within an org. 

1. Is this something you imagine being same for all repos, or you're looking for something more fine grained (ex Repo A has `80%` , Repo B has `75%` etc.)
2. How do you envision revisiting the coverage level for repo B / repo A over time? 
3. Is the friction around "creating a new commit" related to the manual nature of it. In other words, if there was a way to automatically open PRs that edited the repo yaml, based on inputs in Global yaml, would that help?

**Reply by ishaanshah-alation** [reply]
1. I want it to be more fine grained.

2. We want to increase coverage for repos over time. However, different repos have different initial coverage and we want to gradually increase the coverage.

3. We do not want to create a new commit. Ideally, we can have conditionals based on flag names in the global yaml file.
